modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java

```diff
@@ -122,6 +132,24 @@
 )
 public class EditServerMVCActionCommand extends BaseMVCActionCommand {
 
+	public void cleanUpAddToPagePermissions(ActionRequest actionRequest)
```

Comment:
@dantewang 
Why you are creating a new public method here? anyone is using it? If the only usage is to make it easy to invoke in test, we should remove it.

Tina.

Reply:
@tinatian Sorry my bad, I forgot to change the accessibility modifier after moving the method from the `Util` to `EditServerMVCActionCommand`. Will resend.